lib/core/lib/ops/teamOps.js

```diff
@@ -286,7 +286,7 @@ function createDefaultUser (job, next) {
   const { organizationId } = job.params
 
   userOps.insertUser(job.client, { id, name, organizationId }, (err, user) => {
-    if (err) return next(Boom.badImplementation(err))
+    if (err) return next(err)
```

Comment:
@mihaidma you were right, have done a pretty big review of error handling and have made some changes. Can you review again please?